.github/workflows/tests.yml

```diff
@@ -2,6 +2,8 @@ name: tests
 
 on:
   push:
+  repository_dispatch:
+    types: [Tardis-Dispatch]
```

Comment:
should be `tardis-dispatch` right? is the convention alright?

Reply:
yep i will fix this.